apps/web/src/routes/search.tsx

```
@@ -0,0 +1,131 @@
+function SearchResultsPage() {
+  const { q, tags, rating, sort, page } = Route.useSearch();
+  const filters = { tags, minRating: rating, sort };
+
+  const { data, isLoading, isError, error } = useSearchResults({
+    query: q || '',
+    filters,
+    page,
+  });
+
+  const updateFilters = (newFilters: typeof filters) => {
+    const navigate = Route.useNavigate();
```

ESLint

[error] React Hook "Route.useNavigate" is called in function "updateFilters" that is neither a React function component nor a custom React Hook function. React component names must start with an uppercase letter. React Hook names must start with the word "use".

(react-hooks/rules-of-hooks)